ui/component/settingDefaultQuality/view.jsx

```diff
@@ -0,0 +1,70 @@
+// @flow
+import React from 'react';
+import { FormField } from 'component/common/form';
+import { VIDEO_QUALITY_OPTIONS } from 'constants/video';
+import { toCapitalCase } from 'util/string';
+
+type Props = {
+  defaultQuality: ?string,
+  doSetDefaultVideoQuality: (value: ?string) => void,
+};
+
+export default function SettingDefaultQuality(props: Props) {
+  const { defaultQuality, doSetDefaultVideoQuality } = props;
+
+  const [enabled, setEnabled] = React.useState<boolean>(Boolean(defaultQuality));
+
+  const valueRef = React.useRef(VIDEO_QUALITY_OPTIONS[0]);
+
+  function handleEnable() {
+    if (enabled) {
+      setEnabled(false);
+      // From enabled to disabled -> clear the setting
+      doSetDefaultVideoQuality(null);
```

Comment:
<img width="195" alt="image" src="https://user-images.githubusercontent.com/64950861/164136058-b9fd2c7a-2f6e-4cdd-8974-7c6dcaaafe1d.png">

The 2 controls is based on 1 variable, that usually yields confusing UI, although minor.

Example:
1. Set to "720p" + "enabled"
2. Set to "disabled" and save.
3. Return to the Settings Page later.  I expect to be able to just toggle to "enable" and get "720", but the drop-down always reverts to "Auto".

Perhaps just put the "null/disable" case as just another drop-down entry (say, "Off", since 'null' is invalid as a drop-down, as shown in the previous warning), instead of a checkbox.  If want the checkbox, would probably need another variable.